### PR TITLE
Aggregate results 

### LINKED ISSUE #275: chunking without synthetic padding
This code shows a way to divide an SDSS frame into "padded chunks". Each pixel belongs to at most one chunk, but a pixel may be contained in as many as four padded chunks. Pixels on the border of an SDSS frame are not part of any chunk. No synthetic padding is necessary, and the code should run efficiently.

```
ch = 200  # chunk height
cw = 300  # chunk width
cp = 24  # chunk padding

ihic = image.size(0) // ch  - 1  # image height in chunks
iwic = image.size(1) // cw  - 1  # image width in chunks

padded_chunks = torch.zeros((ihic, iwic, ch + 2 * cp, cw + 2 * cp))

for i in range(ihic):
    for j in range(iwic):
        h, w = i * ch + cp, j * cw + cp
        print(h, w)
        padded_chunks[i, j] = image[(h - cp):(h + ch + cp), (w - cp):(w + cw + cp)]

# TODO: 1. infer catalogs for all padded chunks
# TODO: 2. discard catalog entries that are outside the chunk (i.e., that are in the padding)
# TODO: 3. merge the catalogs
# TODO: 4. score the merged catalog (excluding any coadd objects that are not within a chunk)
```

### LINKED ISSUE #276: precision/recall for detections
Following #275, we need to compute precision and recall for objects in one SDSS frame (excluding the border).

Precision is the matched objects divided by the total number of bliss detection.

Recall is the matched objects divided by the total number of coadd detections (excluding coadd objects that are too dim to detect in one frame).

### LINKED ISSUE #277: confusion matrix for star/galaxy classification
Following #275, we can compute a confusion matrix for one frame, for star/galaxy classification.

### PR DESCRIPTION
**Pull request by ismael-mendoza**
- Closes #277 
- Closes #276 
- Closes #275 
- Worst images are plotted in sleep phase reconstruction results. 
- Flag to annotate probabilities in images (star vs galaxy) or not
- cosmetic changes to `sdss-results.ipynb` notebook
- Results can be found in `case_studies/sdss_galaxies/sdss_results.ipynb`

### CODE REVIEW

**Comment by ismael-mendoza** [comment]
Hi @jeff-regier this PR is almost ready to merge, I just had one question. How do I determine which objects are "too dim" in the frame that were found by PHOTO in the coadd?

**Comment by ismael-mendoza** [comment]
Feel free to take a look at the notebooks and the changes in `ReviewNB` there is a link above

**Review by jeff-regier — APPROVED** [review]
Looks like good progress. Some of the code that is notebooks (e.g. looping over chunks) should be part of the bliss package instead. The looping over chunks should get called by the `bliss predict ... ` command, right?

Also the confusion matrix and precision/recall could be stored along with the plotting functions, in a file called reporting.py. That way the can called by other code.

But those changes don't need to stop merging this pr.

**Comment by ismael-mendoza** [comment]
Thanks @jeff-regier I agree, I can potentially move this code to the package rather than have it in a notebook after resolving the issue with the dim galaxies.
